editor/editor_inspector.cpp

```diff
@@ -1102,7 +1102,7 @@ Control *EditorInspectorCategory::make_custom_tooltip(const String &p_text) cons
 				text += "\n" + property_doc;
 			}
 		}
-		help_bit->set_text(text); //hack so it uses proper theme once inside scene
+		help_bit->call_deferred(SNAME("set_text"), text); //hack so it uses proper theme once inside scene
```

Comment:
For the reference, the comment was wrong because @reduz removed the hack in 047e0b7de5c - but as we see here the hack is still needed.